brokenLinkReport.html

```diff
@@ -0,0 +1,20 @@
+<div><h4>Filepath: <a href="/Users/jackryan/Desktop/design-system/README.md" target="_blank">design-system/README.md</a></h4><div style="margin-left:.5rem"><a href="http://facebook.github.io/jest/docs/en/snapshot-testing.html" target="_blank">http://facebook.github.io/jest/docs/en/snapshot-testing.html</a> <p>Code: 404</p></div><br/></div>
```

Comment:
Should we gitignore this file?

Reply:
Yeah, I think for the time being that makes sense. I might add a line to the script that automatically opens this file in the browser.